app/webroot/test.php

```diff
@@ -96,7 +96,7 @@
 	trigger_error("CakePHP core could not be found. Check the value of CAKE_CORE_INCLUDE_PATH in APP/webroot/test.php. It should point to the directory containing your " . DS . "cake core directory and your " . DS . "vendors root directory.", E_USER_ERROR);
 }
 
-if (Configure::read('debug') < 1) {
+if (Configure::read('debug') < CAKE_DEBUG_MODE) {
```

Comment:
Most of the times I use `if (Configure::read('debug') > 0) `

I don't like using the  `if(!Configure::read('debug'))` because it doesn't make clear to the reader of the code what is the return type. It is just a personal taste.

So by replacing this with constants I can track more easily through my IDE where I used it. Same as doing a search for `Configure::read('debug')`. I saw the changes about the autocompletion/docblock etc you are doing now on 3.x and I thought about this MR and give it a try.

But as @dereuromark said it's a deprecated version and it might not worth it.